content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+---
+Title: PDF-Templates
+Weight: 85
+---
+
+
+:toc:
+:toc-title: Table of content
+
+:experimental:
+
+:imagesdir: ./../../../../images/en/user
+
+ifdef::env-github[:imagesdir: ../../../../../master/static/images/en/user/PDFTemplates]
+
+:btn: btn:
+
+ifdef::env-github[:btn:]
+
+= PDF-Template
+
+This module allows you to create document templates. In the future, based on these templates, PDF documents or emails can be created, in which the necessary data from the SuiteCRM will automatically be added during the merge process.
+
+//image:image1.png[PDF-Template]
+
+
+== Creating the PDF-Template
```

Comment:
I suggest removing the hyphen and using just `PDF Template` instead of `PDF-Template`. This would need to be changed in many places (including page title).